indico/modules/events/models/events.py

```diff
@@ -265,6 +265,12 @@ def __table_args__(cls):
         db.ForeignKey('indico.files.id'),
         nullable=True
     )
+    #: The protection setting for speaker submissions
+    speakers_can_submit = db.Column(
```

Comment:
I just realized this name is way too generic. The setting only affects subcontribution speakers.

So it should be `subcontrib_speakers_can_submit`. Before changing this make sure to downgrade your DB revision so you won't have issues upgrading again with the new column name.